### PR TITLE
[TwigBridge] fix for legacy asset() with EmptyVersionStrategy

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15156 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Extension/AssetExtension.php [hunk 1]
```diff
@@ -108,11 +108,15 @@ private function getLegacyAssetUrl($path, $packageName = null, $absolute = false
             $v->setAccessible(true);
             $currentVersionStrategy = $v->getValue($package);
 
-            $f = new \ReflectionProperty($currentVersionStrategy, 'format');
-            $f->setAccessible(true);
-            $format = $f->getValue($currentVersionStrategy);
-
-            $v->setValue($package, new StaticVersionStrategy($version, $format));
+            if (property_exists($currentVersionStrategy, 'format')) {
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/Extension/AssetExtension.php]: Can you add a comment so that we remember to remove this code in master?

Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Extension/AssetExtension.php]: Isn't the whole method subject to be removed in 3.0?

Comment (fabpot) [inline src/Symfony/Bridge/Twig/Extension/AssetExtension.php]: of course, it is :)

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**